src/worker.py

```
@@ -1329,6 +1314,331 @@ async def serve_static(path: str, env):
+        )
+
+        if not still_connected:
+            self._broadcast(json.dumps({
+                "type":           "participant_left",
+                "participant_id": pid,
+                "display_name":   dname,
+            }))
+
+        self._broadcast_room_state()
+
+    async def on_webSocketError(self, ws, error):
```

Ruff (0.15.10)

[warning] Unused method argument: `error`

(ARG002)